lib/ui/views/login/login_view.dart

```diff
@@ -21,7 +21,7 @@ import 'login_viewmodel.dart';
 class LoginView extends StackedView<LoginViewModel> with $LoginView {
   LoginView({Key? key}) : super(key: key);
 
-  final GlobalKey<FormState> _formKey = GlobalKey<FormState>();
+  static final GlobalKey<FormState> _formKey = GlobalKey<FormState>();
```

Comment:
This is going to be a routing blow-up, always check the things you add for a temporary fix.

Reply:
That's the thing, nothing blows up from my end. And that's the only fix for keyboard disappearing as soon as it's called.

https://github.com/flutter/flutter/issues/20042